src/mrack/host.py

```diff
@@ -51,6 +51,7 @@ def host_from_json(host_data):
         host_data["username"],
         host_data["password"],
         host_data["error"],
+        host_data["meta_extra"],
```

Comment:
This should use `host_data.get("meta_extra")` for the cases where e.g. `mrack list` was called with DB created by previous version of mrack. 

Now it causes e.g.:
```
  File "/home/pvoborni/.local/lib/python3.10/site-packages/mrack/dbdrivers/file.py", line 37, in __init__
    self.load()
  File "/home/pvoborni/.local/lib/python3.10/site-packages/mrack/dbdrivers/file.py", line 51, in load
    host = host_from_json(raw_host)
  File "/home/pvoborni/.local/lib/python3.10/site-packages/mrack/host.py", line 54, in host_from_json
    host_data["meta_extra"],
```